src/fragments/lib/auth/js/mfa.mdx

```diff
@@ -40,6 +41,18 @@ Auth.verifyTotpToken(user, challengeAnswer)
   .catch((e) => {
     // Token is not verified
   });
+
+// If signing in with MFA is enabled, use confirmSignIn method to 
+// send a new MFA code to be used with Authenticator. 
```

Comment:
The previous 2 comments in this section start with `first` and `then`, maybe put a `finally` for this one or take out the sequence words completely.

Also, `confirmSignIn` doesn't send the new MFA code to the authenticator, instead you would have already set up your MFA authenticator program and it should be auto-refreshing every ~30 seconds or whatever the time is on the app. `confirmSignIn` is used to verify that you have the MFA code in addition to the username/password. So this method would be used following the `signIn` method which then would fail because it needs the 
SMS_MFA or SOFTWARE_TOKEN_MFA `user.challengeName`